apps/client/src/app/pages/auth-devices/auth-devices-page-routing.module.ts

```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
```

Comment:
I would move `<gf-auth-device-settings>` to `account-page.html` and remove this route